README.md

```diff
@@ -22,3 +29,34 @@ repo forall -p -c 'git checkout $REPO_RREV'
 ```
 ## Build Instructions
 Each component can be built with CMAKE, or use the [tools/maptools.py](tools/maptools.py) build command.
+
+Make sure the dependencies of maptools are installed:
+```
+cd prplMesh/tools
+pip install -r requirements.txt
+```
+
+Build everything:
+```
+./maptools.py build -n -f CMAKE_INSTALL_LIBDIR=lib MSGLIB=zmq BUILD_TESTS=ON CMAKE_BUILD_TYPE=Debug BWL_TYPE=DWPAL TARGET_PLATFORM=linux
```

Comment:
```suggestion
python ./maptools.py build all -n -f CMAKE_INSTALL_LIBDIR=lib MSGLIB=zmq BUILD_TESTS=ON CMAKE_BUILD_TYPE=Debug BWL_TYPE=DWPAL TARGET_PLATFORM=linux
```

Reply:
Do we want to instruct people to build for containers by default? Why not add all options in the readme?

### Build and run local agent & controller over UDS
// dummy mode instructions
### Build and run local agent & controller and remote agent over ZMQ/NNG (requires 2 machines)
// Normal instructions - using 2 machines
### Build and run local agent & controller and remote agent using containers 
// How to run with containers

Reply:
Yes that should be added, independent of this branch.